### ISSUE TITLE
Additional step needed for this to work

### ISSUE THREAD

**Issue by FierceMuppet** [MicrosoftDocs/architecture-center#3672]
I used this guide and tried several times, and it did not work.  After troubleshooting for quite a while, I ended up having to link the private DNS to the Vnet after I set up the private link.  Please update the guide.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 7499b146-3340-d7d1-6c5f-3ed508b773cb
* Version Independent ID: 61b10612-1136-3840-c7c1-804e63b23410
* Content: [Web app private connectivity to Azure SQL Database - Azure Example Scenarios](https://learn.microsoft.com/en-us/azure/architecture/example-scenario/private-web-app/private-web-app)
* Content Source: [docs/example-scenario/private-web-app/private-web-app.yml](https://github.com/microsoftdocs/architecture-center/blob/main/docs/example-scenario/private-web-app/private-web-app.yml)
* Service: **architecture-center**
* Sub-service: **example-scenario**
* GitHub Login: @jelledruyts
* Microsoft Alias: **ssumner**

**Comment by jelledruyts** [comment]
Hi @FierceMuppet, thanks for reaching out. How did you create the private endpoint exactly? If you use the Azure Portal following the instructions in the guide, then the private DNS zone gets linked to the VNet automatically and no extra step should be necessary (I just checked again to be absolutely sure). If you are using the Portal and that's not happening for you then it sounds like an issue with your environment perhaps, and a support case might be needed to figure out why that's not happening for you.

**Comment by FierceMuppet** [comment]
No, I followed the steps EXACTLY. I even had a Senior Network Engineer from Microsoft look at it with me.  The problem was not creating the private endpoint nor associating the app service to the subnet.  The app service was finding the SQL Server.  The name resolver was still resolving to the external IP Address. I had to go to the private DNS – privatelink.database.windows.net and add the virtual network link manually.  After that, it worked.

**Comment by jelledruyts** [comment]
Ok thanks for confirming what you've done. As I said, this is definitely not the expected behavior and I've tried it again today to be sure and did not have that problem. If you can reproduce it I would suggest to open a support case! Now unless this turns out to be a general problem that is affecting many people I don't think updating the article to include steps a customer _shouldn't_ have to do would help others at this time, so I'm not planning to update the article at this time.

**Comment by FierceMuppet** [comment]
I could reproduce several times.  I have done my due diligence and made you aware of this.  No need to open a ticket on my end as I can configure it to work on my end.

On your end was the private dns linked to your vnet when you performed this?

**Comment by jelledruyts** [comment]
That's right, I created the private endpoint for SQL and the private DNS zone was automatically linked to the VNet.

**Comment by FierceMuppet** [comment]
Hmm.  That is the issue.  Did you do this from scratch? Permissions possibly?  My role is Contributor.  Have you tried with a user with that role and not owner or admin?

**Comment by FierceMuppet** [comment]
OK, take care.  I am going to script this out with PowerShell for future deployment and I will see if it has the same issue.
